<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlBrowserWidget</class>
 <widget class="QWidget" name="SqlBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eric6 SQL Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="SqlConnectionWidget" name="connections">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QTableView" name="table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="queryGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>SQL Query</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="sqlEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>18</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Enter the SQL query to be executed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="toolTip">
           <string>Press to clear the entry</string>
          </property>
          <property name="text">
           <string>&amp;Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="executeButton">
          <property name="toolTip">
           <string>Press to execute the query</string>
          </property>
          <property name="text">
           <string>&amp;Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="toolTip">
    <string>Inserts a new row</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete Row</string>
   </property>
   <property name="statusTip">
    <string>Deletes the current row</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SqlConnectionWidget</class>
   <extends>QTreeView</extends>
   <header>.SqlConnectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sqlEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>executeButton</tabstop>
  <tabstop>connections</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
